asyncio.wait_for(coro, timeout)

await must have!!!

Запуск задачи: когда вы вызываете asyncio.wait_for(), она начинает выполнение переданной coro или Future.

Ожидание завершения: asyncio.wait_for() ожидает завершения задачи в течение указанного таймаута. В это время другие
 асинхронные задачи могут продолжать выполняться, поскольку asyncio.wait_for() не блокирует цикл событий.

Проверка таймаута: Если задача завершается до истечения таймаута, asyncio.wait_for() возвращает результат задачи.
 Если задача не завершается в течение таймаута, asyncio.wait_for() автоматически отменяет задачу и генерирует исключение
  asyncio.TimeoutError.

Обработка исключений: Если во время выполнения задачи возникает исключение, asyncio.wait_for() пробросит его дальше.
 Это означает, что вы должны быть готовы обрабатывать исключения, которые могут возникнуть во время выполнения вашей задачи.
